test/GasUsage.test.js

```diff
@@ -40,6 +40,20 @@ describe('ERC721A Gas Usage', function () {
     });
   });
 
+  it('mintOneERC2309', async function () {
+    let contract = await deployContract('ERC721AWithERC2309Mock', ['Azuki', 'AZUKI']);
+    await contract.mintOneERC2309(this.owner.address);
```

Comment:
This test case is not following the ERC-721 standard.

Please update all test cases to only demonstrate uses cases with are within the project scope, i.e. fully supporting the ERC-721 standard.